lesson_8/funstions.js

```diff
@@ -0,0 +1,119 @@
+/*Дано масив з елементами різних типів. Створити функцію яка вираховує середнє арифметичне лише числових елементів даного масиву.*/
+
+numbers = [1, 2, 3, 14, 'bla', 3.2]
+
+/**
+ * @param {array <any>} nums 
+ * 
+ * @returns array <number>
+ */
+
+
+function average(nums) {
+  result = [];
+  for (let i = 0; i < nums.length; i++) {
+    if (Number(nums[i])) {
+      result.push(nums[i]);
+    }
+  }
+  return result.reduce((a, b) => (a + b)) / result.length;
```

Comment:
dont forget initial argument
```suggestion
  return result.reduce((a, b) => (a + b), 0) / result.length;
```